Assets/Scripts/Player/PlayerInput.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
```

Comment:
It might make more sense for the player input script to not be attached to the player since you are destroying the player when their heath is zero and you will probably still want the user to be able to interact with the game after the player has been destroyed. While it might not be an ideal solution, you could create a singleton wrapper around the auto-generated player input script and use that instead. Something like [this](https://github.com/KerbalJeb/Icarus/blob/master/Assets/Scripts/InputManager.cs)